android/src/ti/notificationbanner/TitaniumNotificationBannerModule.java

```diff
@@ -53,6 +58,8 @@ public void show(KrollDict args)
 		Alerter.create(currentActivity)
 			.setTitle(title)
 			.setText(subtitle)
+			.setDuration(duration*1000) //iOS parity. Alerter expects miliseconds
+			.setBackgroundColorInt(color)
```

Comment:
Same here

Reply:
Which should be the default values for duration and backgroundColor?